apps/condo/domains/property/components/panels/Builder/BuildingPanelEdit.tsx

```diff
@@ -534,11 +539,13 @@ const ChessBoard: React.FC<IChessBoardProps> = (props) => {
                                                 section={section}
                                                 builder={builder}
                                                 refresh={refresh}
+                                                duplicatedUnitIds={duplicatedUnitIds}
```

Comment:
Why you need to pass id's to the section component? That's not make sense. Anyway you are not used this prop
```suggestion
```